### PR TITLE
pherry: set 4 to default longevity

### PR DESCRIPTION
**Pull request by h4x3rotab**
This can reduce the risk of tx stuck.

### CODE REVIEW

File: standalone/pherry/src/main.rs [hunk 1]
```diff
@@ -172,10 +172,11 @@ struct Args {
     )]
     tip: u64,
     #[structopt(
+        default_value = "5",
```
Comment (kvinwang) [inline standalone/pherry/src/main.rs:175]: Why `5` rather than `4`?  It should be a power of two.

Comment (h4x3rotab) [reply standalone/pherry/src/main.rs:175]: Got it. We need to add more check there.

File: standalone/pherry/src/main.rs [hunk 2]
```diff
@@ -985,6 +986,8 @@ fn preprocess_args(args: &mut Args) {
         args.use_dev_key = true;
         args.mnemonic = String::from("//Alice");
     }
+    assert!(args.longevity == 0 || args.longevity >= 4, "Option --longevity must be 0 or >= 4.");
+    assert_eq!(args.longevity.count_ones(), 1, "Option --longevity must be power of two.");
```
Comment (kvinwang) [inline standalone/pherry/src/main.rs:990]: 0 will assert fail?

**Review by kvinwang — APPROVED** [review]
LGTM

**Pull request merged**